test/module/irohad/multi_sig_transactions/CMakeLists.txt

```diff
@@ -35,3 +35,9 @@ target_link_libraries(mst_processor_test
     mst_processor
     logger
     )
+
+AddTest(gossip_propagation_strategy_test gossip_propagation_strategy_test.cpp)
```

Comment:
Fix code style

Reply:
What's wrong?

Reply:
I expect something like this:
```add_library(mst_transport
        impl/mst_transport_grpc.cpp
        )
```
Where each impl file on own line and close bracket also on own. This is useful when reader looks on diff. Same idea with end lines also.